app/src/main/java/org/mozilla/tv/firefox/settings/SettingsFragment.kt

```diff
@@ -111,12 +112,21 @@ class SettingsFragment : Fragment() {
     ): View {
         val view = inflater.inflate(R.layout.settings_screen_fxa_profile, parentView, false)
 
+        setupFxaText(view)
         setupFxaProfileListeners(view)
         profileDisposable = observeFxaProfile(view)
 
         return view
     }
 
+    private fun setupFxaText(view: View) {
+        val appName = resources.getString(R.string.app_name)
+        view.button_firefox_tabs.text = resources.getString(R.string.fxa_settings_primary_button, appName)
+        // Username is positioned and styled differently, so it is left blank here
+        // and set on another TextView
+        view.description_top.text = resources.getString(R.string.fxa_settings_body, "")
+    }
+
     private fun setupFxaProfileListeners(view: View) {
```

Comment:
nit: `setupFxaProfileClickListeners`. Otherwise, it's easy to confuse with `observeFxaProfile`.